pkg/api/types.go

```diff
@@ -8,13 +8,13 @@ const (
 // OpenShiftManagedCluster complies with the ARM model of resource definition in
 // a JSON template.
 type OpenShiftManagedCluster struct {
-	Plan       *ResourcePurchasePlan `json:"plan,omitempty"`
-	Properties *Properties           `json:"properties,omitempty"`
-	ID         string                `json:"id,omitempty"`
-	Name       string                `json:"name,omitempty"`
-	Type       string                `json:"type,omitempty"`
-	Location   string                `json:"location,omitempty"`
-	Tags       map[string]string     `json:"tags"`
+	Plan       ResourcePurchasePlan `json:"plan,omitempty"`
+	Properties Properties           `json:"properties,omitempty"`
+	ID         string               `json:"id,omitempty"`
+	Name       string               `json:"name,omitempty"`
+	Type       string               `json:"type,omitempty"`
+	Location   string               `json:"location,omitempty"`
+	Tags       map[string]string    `json:"tags"`
 
 	Config *Config `json:"config,omitempty"`
```

Comment:
I was touching only fields involved in the current conversions. This could be updated as part of the admin api changes? Less work for this PR which is already massive.

Reply:
Fine.